packages/patternfly-4/react-core/src/components/Dropdown/Dropdown.js

```diff
@@ -63,16 +100,33 @@ class Dropdown extends React.Component {
           isPlain && styles.modifiers.plain,
           direction === DropdownDirection.up && styles.modifiers.top,
           position === DropdownPosition.right && styles.modifiers.alignRight,
-          isOpen && styles.modifiers.expanded,
+          isOpen && focusedItemRef && styles.modifiers.expanded,
           className
         )}
         ref={ref => {
           this.parentRef = ref;
```

Comment:
refs should be using the new syntax `React.createRef()` and then they are accessed through `this.parentRef.current` https://reactjs.org/docs/refs-and-the-dom.html#creating-refs we are only supporting the latest react right now so it should be safe to use.